util/endlessh@.service

```diff
@@ -0,0 +1,45 @@
+[Unit]
+Description=Endlessh SSH Tarpit
+Documentation=man:endlessh(1)
+Requires=network-online.target
```

Comment:
fwiw the only thing you need to have `systemctl start endlessh@2222` work is `Requires=endlessh@%i.socket` here -- systemd will automatically pull the socket in then.